external/kafka-0-10-sql/src/test/resources/log4j.properties

```diff
@@ -25,4 +25,6 @@ log4j.appender.file.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss.SSS} %t %p %c{
 
 # Ignore messages below warning level from Jetty, because it's a bit verbose
 log4j.logger.org.spark-project.jetty=WARN
-
+log4j.logger.org.apache.spark.sql.kafka010.KafkaTestUtils=DEBUG
+log4j.logger.org.apache.directory.server.kerberos.kdc.authentication=DEBUG
+log4j.logger.org.apache.directory.server.core.DefaultDirectoryService=DEBUG
```

Comment:
Another quick question. How much noisy is it? I haven't tried it by myself yet but seems a bit overkill.

Reply:
Without these 3 loggers the output is `2836` lines and with them `3116`. I think it's a feasible addition. Having no/missing spots in the debug logs it's pretty hard to debug failures.

Reply:
Thanks. Since this is a test resource, it looks safe to me.